docs/userguide/modules/binding-workloads-using-sbo/pages/unbinding-workloads.adoc

```diff
@@ -0,0 +1,35 @@
+[#unbinding-workloads]
+= Unbinding workloads
+
+You can unbind a workload from a backing service by using the `kubectl` or `oc` command line tool.
+
+* To unbind a workload from a backing service, delete the `ServiceBinding` custom resource (CR) linked to it:
++
+[source,terminal]
+----
+$ kubectl delete ServiceBinding <.metadata.name>
+----
++
+OR
++
+[source,terminal]
+----
+$ oc delete ServiceBinding <.metadata.name>
+----
++
+where `<.metadata.name>` is the name of the `ServiceBinding` resource.
++
+.Examples
```

Comment:
```suggestion
where `<.metadata.name>` is the name of the `ServiceBinding` resource.
+
[NOTE]
====
`ServiceBinding` custom resource (CR) belongs to the `binding.operators.coreos.com/v1alpha1` API. To unbind a workload from a backing service bound using a `ServiceBinding` custom resource (CR) from the `servicebinding.io/v1alpha3` API:

[source,terminal]
----
$ kubectl delete servicebindings.servicebinding.io <.metadata.name>
----

or

[source,terminal]
----
$ oc delete servicebindings.servicebinding.io <.metadata.name>
----
====
+
.Examples
```

Similarly as we mention both APIs in the `Binding workloads using Service Binding Operator` section, we should mention both APIs here, especially when the command is different for each.

Reply:
Fixed

Reply:
Fixed